### ISSUE TITLE
Integration test: aws error

### ISSUE THREAD

**Issue by dkoshkin** [apprenda/kismatic#57]
```
Happy Path Installation Tests Installing with package installation enabled Targeting AWS infrastructure using a 1/1/1 layout with Ubuntu 16.04 LTS 
  should result in a working cluster
  /var/snap-ci/src/github.com/apprenda/kismatic/integration/framework.go:16

STEP: Provisioning nodes
AWS NODE "i-0e65bac5a83984725" MUST BE CLEANED UP MANUALLY


• Failure [10.245 seconds]
Happy Path Installation Tests
/var/snap-ci/src/github.com/apprenda/kismatic/integration/install_test.go:194
  Installing with package installation enabled
  /var/snap-ci/src/github.com/apprenda/kismatic/integration/install_test.go:93
    Targeting AWS infrastructure
    /var/snap-ci/src/github.com/apprenda/kismatic/integration/install_test.go:92
      using a 1/1/1 layout with Ubuntu 16.04 LTS
      /var/snap-ci/src/github.com/apprenda/kismatic/integration/install_test.go:75
        should result in a working cluster [It]
        /var/snap-ci/src/github.com/apprenda/kismatic/integration/framework.go:16

        Expected error:
            <*awserr.requestError | 0xc420376690>: {
                awsError: {
                    code: "InvalidInstanceID.NotFound",
                    message: "The instance ID 'i-0e65bac5a83984725' does not exist",
                    errs: nil,
                },
                statusCode: 400,
                requestID: "29729994-4cfb-404a-ba4e-6cc4268f010d",
            }
            InvalidInstanceID.NotFound: The instance ID 'i-0e65bac5a83984725' does not exist
                status code: 400, request id: 29729994-4cfb-404a-ba4e-6cc4268f010d
        not to have occurred

        /var/snap-ci/src/github.com/apprenda/kismatic/integration/framework.go:39
```
```
Happy Path Installation Tests Installing with package installation enabled Targeting AWS infrastructure using a 3/2/3 layout with CentOS 7 
  should result in a working cluster
  /var/snap-ci/src/github.com/apprenda/kismatic/integration/framework.go:16

STEP: Provisioning nodes
..

• Failure [23.342 seconds]
Happy Path Installation Tests
/var/snap-ci/src/github.com/apprenda/kismatic/integration/install_test.go:194
  Installing with package installation enabled
  /var/snap-ci/src/github.com/apprenda/kismatic/integration/install_test.go:93
    Targeting AWS infrastructure
    /var/snap-ci/src/github.com/apprenda/kismatic/integration/install_test.go:92
      using a 3/2/3 layout with CentOS 7
      /var/snap-ci/src/github.com/apprenda/kismatic/integration/install_test.go:91
        should result in a working cluster [It]
        /var/snap-ci/src/github.com/apprenda/kismatic/integration/framework.go:16

        Expected error:
            <*awserr.requestError | 0xc420323b00>: {
                awsError: {
                    code: "MissingParameter",
                    message: "The request must contain the parameter InstanceId",
                    errs: nil,
                },
                statusCode: 400,
                requestID: "5061efcb-960f-49a3-ba2e-7d865e90ac2d",
            }
            MissingParameter: The request must contain the parameter InstanceId
                status code: 400, request id: 5061efcb-960f-49a3-ba2e-7d865e90ac2d
        not to have occurred

        /var/snap-ci/src/github.com/apprenda/kismatic/integration/framework.go:39
```

**Comment by dasmb** [comment]
In CloudTrail, this looks like a race condition. 
<img width="1581" alt="screen shot 2016-11-08 at 3 52 42 pm" src="https://cloud.githubusercontent.com/assets/5652591/20122299/9b7e00b4-a5cb-11e6-9717-9d154098ffcc.png">

And sure enough:
```The specified instance does not exist. Ensure that you have indicated the region in which the instance is located, if it's not in the default region. This error may occur because the ID of a recently created instance has not propagated through the system. For more information, see Eventual Consistency.```

```To manage eventual consistency, you can do the following:

Confirm the state of the resource before you run a command to modify it. Run the appropriate Describe command using an exponential backoff algorithm to ensure that you allow enough time for the previous command to propagate through the system. To do this, run the Describe command repeatedly, starting with a couple of seconds of wait time, and increasing gradually up to five minutes of wait time.
Add wait time between subsequent commands, even if a Describe command returns an accurate response. Apply an exponential backoff algorithm starting with a couple of seconds of wait time, and increase gradually up to about five minutes of wait time.```

**Comment by alexbrand** [comment]
Closing as this should be addressed by #58
